test/hack/bin/check-opsrecipes.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+set -euo pipefail
+
+# List of generated rules
+RULES_FILES=(./test/hack/output/*/prometheus-rules/templates/alerting-rules/*)
+#RULES_FILES=(./test/hack/output/*/prometheus-rules/templates/alerting-rules/up*)
+
+DEBUG_MODE=false
+CHECK_EXTRADATA_ERRORS=false
+CHECK_NORECIPE_ERRORS=true
+CHECK_UNEXISTINGRECIPE_ERRORS=true
+
+# Parameters:
+# - an element
+# - an array
+# Returns:
+# - 0 if element is not found in the array
+# - 1 if element is found in the array
+isInArray () {
+    referenceElement="$1" && shift
+    array=("$@")
+
+    for element in "${array[@]}"; do
+        [[ "$element" == "$referenceElement" ]] \
+            && return 0
+    done
+    return 1
+}
+
+listOpsRecipes () {
+    # find list of opsrecipes from git repo
+    tmpDir="$(mktemp -d)"
+    git clone --depth 1 --single-branch -b main -q git@github.com:giantswarm/giantswarm.git "$tmpDir"
+    find "$tmpDir"/content/docs/support-and-ops/ops-recipes -name \*.md \
```

Comment:
It always bugs me when not providing a type filter to find, how about
```suggestion
    find -type f "$tmpDir"/content/docs/support-and-ops/ops-recipes -name \*.md \
```

Reply:
Sure.
Although `-type` has to be after the path. :wink: